pokemonterminal/slideshow.py

```diff
@@ -0,0 +1,61 @@
+import atexit
+import random
+import sys
+
+from .platform.named_event import create_named_event
+
+def __print_fork(pid, length, delay):
+    print(f"Starting slideshow with {length} Pokemons and a delay of {delay} minutes.")
+    print(f"Forked process to background with PID {pid}.")
+    print("You can stop it with 'pokemon -c'. (add '-w' if this is a wallpaper slideshow)")
+
+def __mac_exit_listener(event):
+    event.wait()
+
+def __get_sleeper_and_listener(event):
+    if sys.platform == 'darwin':
+        from threading import Thread
+        t = Thread(target=__mac_exit_listener, args=(event, ))
```

Comment:
why do we need a separate thread only on mac?

Reply:
On Windows and Linux we use is_set in the while loop guard, which is not available on mac.